### DISCUSSION TITLE
How to disable "quiet" globally?

### DISCUSSION THREAD

**Discussion by gcb** [netblue30/firejail#6537] [Q&A] [answered]
This is a follow up on https://github.com/netblue30/firejail/discussions/6535

I decided I do not like, at all, the quiet option to exist on any profile. Is there a way to disable it entirely?

**Comment by kmk3** [comment]
Edit: This is wrong; see the [next comment](https://github.com/netblue30/firejail/discussions/6537#discussioncomment-11198351).

> How to disable "quiet" globally?

Add the following to /etc/firejail/globals.local:

```
ignore quiet
```

**Reply by rusty-snake** [reply, marked as answer]
@kmk3 `globals.local` is include _after_ `quiet`. We have an issue for that, somewhere.

Edit, here it is: #3503

**Comment by gcb** [comment]
Should we have a way to "document via comments" things for globals.local? maybe ship one with all the options commented out?

**Reply by gcb** [reply]
@rusty-snake firejail files already do that for everything. Almost all `.profile` files have lots of comments of what to use on the `.local` ones. I'm thinking we should have a `globals.profile` or `globals.local` on the standard distribution with all the available options commented out with short explanations. Exactly how it already happens on all the `.profile`s.

**Reply by rusty-snake** [reply]
> Almost *all* .profile files have lots of comments of what to use on the .local ones.

```console
$ grep -E '^#.*\.local' /etc/firejail/*.profile -l | wc -l
561
$ grep -E '^#.*\.local' /etc/firejail/*.profile -L | wc -l
737
```

> firejail files already do that for everything. Almost all .profile files have lots of comments of what to use on the .local ones. 

The most of them are either
- enable foobar for hardening if your system/workflow meets condition abc, or
- ignore foobar if you want/need uncommon feature foobaz.

Both a profile/program specific.

> I'm thinking we should have a globals.profile or globals.local on the standard distribution with all the available options commented out with short explanations.

So what do you want? Best with example.
Every command listed in `man 5 firejail-profile`? i.e. global hardening

Can you give a example of 5 commented options.

**Reply by gcb** [reply]
We are agreeing in a way :) 

what i'm putting up for discussion is a globals.local with the "if you want/need uncommon feature foobaz." comments.
